cmd/soroban-cli/src/commands/contract/mod.rs

```diff
@@ -165,3 +174,16 @@ pub enum SpecOutput {
     /// Pretty print of contract spec entries
     Docs,
 }
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, clap::ValueEnum, Default)]
+pub enum InfoOutput {
+    /// Pretty-printed output of the info entry
+    #[default]
+    Pretty,
```

Comment:
`pretty` is ambiguous. For the `interface` subcommand it is Rust code. For the other subcommands it is human readable unstructured text. I think we should use a different enum for each of the three commands and use more specific names for the formats that align 1-1 with each.

So for the `interface` subcommand accept the value, and use as default, `rust`.

For the other subcommands I'd probably use the term `text`.